src/main/java/seedu/eduke8/exception/ExceptionMessages.java

```diff
@@ -0,0 +1,18 @@
+package seedu.eduke8.exception;
+
+public class ExceptionMessages {
+
+    public static final String ERROR_STORAGE_FAIL = "Oh no! An error has occurred when accessing the file."
+            + "Please check again!";
+    public static final String ERROR_UNKNOWN = "Oh no! An unknown error has occurred.";
+    public static final String ERROR_UNRECOGNIZED_COMMAND = "Oops! The command cannot be understood.\n"
+            + "Please use the \"help\" to get the list of available commands. ";
+    public static final String ERROR_QUIZ_WRONG_FORMAT = "Invalid command! The command for quiz is as follows:\n"
+            + "quiz t/<topic> n/<number of questions>";
+    public static final String ERROR_QUIZ_COMMAND_NOT_IMPLEMENTED = "Oh! This command has yet to be implemented. "
+            + "Stay tune for Version 2.0!";
```

Comment:
Can add a new line between the two parts but great job nonetheless!

Reply:
Okie!